mountpoint-s3-fs/examples/mount_from_config.rs

```diff
@@ -193,19 +169,19 @@ fn load_config<P: AsRef<Path>>(path: P) -> Result<ConfigOptions> {
 
 /// Processes a manifest and stores database in `database_directory`
 fn process_manifests(config: &ConfigOptions, database_directory: &Path) -> Result<Manifest> {
-    // Error if no channels
-    if config.channels.len() != 1 {
-        return Err(anyhow!("Exactly one channel must be configured"));
+    // TODO: fail fast if buckets are in different regions?
+    if config.channels.is_empty() || config.channels.len() > 20 {
+        return Err(anyhow!(
+            "At lest one channel must be specified and the number of channels must not exceed 20"
+        ));
     }
 
-    let channel = &config.channels[0];
-    let csv_path = &channel.manifest_path;
     // Generate manifest path and check if it exists
     let db_path = database_directory.join("metadata.db");
-    info!("Creating manifest for channel {}", channel.directory_name);
+    info!("Ingesting CSV manifests to the DB, channels {:?}", config.channels);
```

Comment:
nit: into the DB

Reply:
let's also review whether we use "DB", "database", or something more specific, e.g. "metadata store"?